classes/class-p4-master-site.php

```diff
@@ -166,6 +166,7 @@ function( $timeout ) {
 		add_filter( 'login_headerurl', [ $this, 'add_login_logo_url' ] );
 		add_filter( 'login_headertitle', [ $this, 'add_login_logo_url_title' ] );
 		add_action( 'login_enqueue_scripts', [ $this, 'add_login_stylesheet' ] );
+		add_action( 'cmb2_after_form', [ $this, 'cmb2_after_form_do_js_validation'], 10, 2 );
```

Comment:
from `phpcs`:

```
Missing space before array closer.
```

Reply:
Issue fixed. @comzeradd could you please check now.